portfolio/src/main/webapp/script.js

```diff
@@ -45,4 +45,12 @@ async function showServerTime() {
   
     const dateContainer = document.getElementById('date-container');
     dateContainer.innerText = textFromResponse;
-  }
+}
+
+async function getServerGreeting() {
+    
+    const responseFromServer = await fetch('/greeting');
+    const g = await responseFromServer.text();
```

Comment:
need to use .json()

Reply:
Updated. Line 53 ow changes to =await responseFromServer.json();